### PR TITLE
fix: Disallow `ComboBox` popup under translucent status bar

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/10339

## PR Type

What kind of change does this PR introduce?

- Bugfix

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/feature-flags.md [hunk 1]
```diff
@@ -35,6 +35,16 @@ By default, Android has a limit of 200% for the text scaling which is not the ca
 
 Use `Uno.UI.FeatureConfiguration.Font.MaximumTextScaleFactor` to control this. 
 
+## ComboBox
+
+### Default preferred placement
+
+By default, `ComboBox` popup is placed in such a way that the currently selected item is centered abocve the `ComboBox`. If you want to adjust this behavior on non-Windows targets, set the `DefaultDropDownPreferredPlacement` property.
```
Comment (Youssef1313) [inline doc/articles/feature-flags.md:42]:
```suggestion
By default, `ComboBox` popup is placed in such a way that the currently selected item is centered above the `ComboBox`. If you want to adjust this behavior on non-Windows targets, set the `Uno.UI.FeatureConfiguration.ComboBox.DefaultDropDownPreferredPlacement` property.
```

File: doc/articles/feature-flags.md [hunk 2]
```diff
@@ -35,6 +35,16 @@ By default, Android has a limit of 200% for the text scaling which is not the ca
...
+
+### Allow popup under translucent status bar
+
+By default, the `ComboBox` popup will not extend under the status bar even if it is set as translucent. If you want to change this behavior, set the `AllowPopupUnderTranslucentStatusBar` property to `true`.
```
Comment (Youssef1313) [inline doc/articles/feature-flags.md:46]:
```suggestion
By default, the `ComboBox` popup will not extend under the status bar even if it is set as translucent. If you want to change this behavior, set the `Uno.UI.FeatureConfiguration.ComboBox.AllowPopupUnderTranslucentStatusBar` property to `true`.
```

File: src/Uno.UI/FeatureConfiguration.cs [hunk 3]
```diff
@@ -81,6 +81,12 @@ public static class ComboBox
 			/// referencing the ** type ** ComboBox in any way.
 			/// </remarks>
 			public static Uno.UI.Xaml.Controls.DropDownPlacement DefaultDropDownPreferredPlacement { get; set; } = Uno.UI.Xaml.Controls.DropDownPlacement.Auto;
+
+			/// <summary>
+			/// Gets or sets a value indicating whether the combobox popup should be allowed
```
Comment (Youssef1313) [inline src/Uno.UI/FeatureConfiguration.cs:86]:
```suggestion
			/// Gets or sets a value indicating whether the ComboBox popup should be allowed
```

File: src/Uno.UI/FeatureConfiguration.cs [hunk 4]
```diff
@@ -81,6 +81,12 @@ public static class ComboBox
...
+			/// to be displayed under translucent status bar on Android. Defaults to false.
+			/// </summary>
+			public static bool AllowPopupUnderTranslucentStatusBar { get; set; }
```
Comment (Youssef1313) [inline src/Uno.UI/FeatureConfiguration.cs:89]: Should we wrap this in `#if __ANDROID__`?

**Review by jeromelaban — APPROVED** [review]

**Review by Youssef1313 — APPROVED** [review]

**Pull request merged**